src/map/Map.js

```diff
@@ -120,7 +120,15 @@ export var Map = Evented.extend({
 
 		// @option trackResize: Boolean = true
 		// Whether the map automatically handles browser window resize to update itself.
-		trackResize: true
+		trackResize: true,
+
+		// @option darkMode: Boolean = false
+		// Force Dark Mode of Leaflet
+		darkMode: false,
+
+		// @option useUserColorScheme: Boolean = true
+		// Use the user color scheme to switch between Light and Dark Mode
+		useUserColorScheme: true

```

Comment:
Perhaps we could do multiple behavior here instead? For example, `'dark' | 'light' | 'auto'`?

Reply:
Good idea!

```javascript
const prefersDarkTheme = window.matchMedia('(prefers-color-scheme: dark)');
```